src/index.d.ts

```diff
@@ -0,0 +1,2248 @@
+// Type definitions for react-native-elements 0.16
+// Project: https://github.com/react-native-training/react-native-elements#readme
+// Definitions by: Kyle Roach <https://github.com/iRoachie>
+//                 Ifiok Jr. <https://github.com/ifiokjr>
```

Comment:
also we should consider adding this to the docs to let users know that it exists and how they can use it if they are beginners in typescript. 

How about we do a fast follow after v0.18 release to add docs for it? thanks.

Reply:
Previously this definition was on DefinitelyTyped which required an extra step after `npm i react-native-elements`. By moving the declaration here, no extra setup is needed

Reply:
>  add docs for it

Reply:
Oooo alright then guess I'll do that